### PR TITLE
refactor : (multiple) major update

### PR DESCRIPTION
**Pull request by orville-wright**
Modernize, polish, pretty. Remove all refs to Legacy Archived products. Update to v4.0 components. Correct contact info. Accurate links. Signed-off-by David Brace <orville.wright@yahoo.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,64 @@
...
+We have a very large [active community](https://github.com/openebs/community), and many storage users [contribute to our product](https://github.com/openebs/community/blob/develop/CONTRIBUTING.md) with discussions, ideas, Issues, Feature requests and even code contribnutions.  
+
+There are many ways to get in touch with our team with our team.
+- **EMAIL us**:  cncf-openebs-maintainers@lists.cncf.io <BR>
+- **Contact the Maintainers** : The up to date Maintainers list [is here](/openebs/MAINTAINERS.md)
+> Reach out via GitHuib to the OpenEBS core leadership team. <BR>
+> :rocket: &nbsp; Ed Robinson | @edrob999 <BR>
+> :star: &nbsp; David Brace | @orville-wright <BR>
+> :zap: &nbsp; Vishnu Attur | @avishnu <BR>
+> :sunglasses: &nbsp; Tiago Castor | @tiagolobocastro <BR>
```
Comment (avishnu) [inline README.md:54]: Typo: Castro

File: README.md [hunk 2]
```diff
@@ -1,43 +1,64 @@
...
+> :floppy_disk: &nbsp; We have +49 Million Volumes deployed globally <BR>
+> :tv: &nbsp; We have +8 Million Global installations <BR>
+> :zap: &nbsp; 1 Million OpenEBS K8s Containers are spawned per week <BR>
+> :sunglasses: &nbsp; 1.1 Million global users <BR>
+
 <BR>
 
+We have a very large [active community](https://github.com/openebs/community), and many storage users [contribute to our product](https://github.com/openebs/community/blob/develop/CONTRIBUTING.md) with discussions, ideas, Issues, Feature requests and even code contribnutions.  
+
+There are many ways to get in touch with our team with our team.
```
Comment (avishnu) [inline README.md:47]: "with our team" repeated.

File: README.md [hunk 3]
```diff
@@ -1,43 +1,64 @@
...
+OpenEBS is a Kubernetes provides Persistent Stateful Storage Platform that has **5 core Data Engines**<BR>
+Each Storage Engine provides differnt Capabilities, Flexibility, Resilliance, Data Protections, and Performance features.<BR>
+
+| ID  | Data-Eegines       | Where to create your issues                            |
+| :---: | :---             | :---                                                   |
+|     | ```Replicated PV```          | ```Replicated storage and data volumes```     |
+|  1  | [Replicated PV Mayastor](https://github.com/openebs/mayastor)      | Distributed vSAN Fabric attached volumes that are replicated   | 
+| &nbsp;                        |    |
+|     | ```Local PV```          | ```Non-replicated node local storage and volumes```   |                                                            |
+|  2. |  [LocalPV-HostPath](https://github.com/openebs/dynamic-localpv-provisioner)     | Dynamically provisioned Node-local volumes with **```HostPath```** resident backend data |
```
Comment (avishnu) [inline README.md:26]: Local PV Hostpath

File: README.md [hunk 4]
```diff
@@ -1,43 +1,64 @@
...
+|  3. |  [LocalPV-ZFS](https://github.com/openebs/zfs-localpv)      | Dynamically provisioned Node-local volumes with an integrated **```ZFS```** storage backend |
```
Comment (avishnu) [inline README.md:27]: Local PV ZFS

File: README.md [hunk 5]
```diff
@@ -1,43 +1,64 @@
...
+|  4. |  [LocalPV-LVM](https://github.com/openebs/lvm-localpv)      | Dynamically provisioned Node-local volumes with an integrated **```LVM```** storage backend |
```
Comment (avishnu) [inline README.md:28]: Local PV LVM

File: README.md [hunk 6]
```diff
@@ -1,43 +1,64 @@
...
+|  5. |  [LocalPV-Raw-device-File](https://github.com/openebs/rawfile-localpv) | Dynamically provisioned Node-local volumes via **```soft-lun RAW Device files```** on ```Hostpath``` resident backend filesystem  |
```
Comment (avishnu) [inline README.md:29]: I'd say, let's not advertise this now, our technical documentation doesn't mention about this as well.
We could take this in once we make the necessary changes?

File: README.md [hunk 7]
```diff
@@ -1,43 +1,64 @@
...
+> - Enterprise Grade data management capabilities such as **snapshots, clones, replicated volumes, DIskGroups, Volume Groups, Aggregates, RAID** <BR>
 <BR>
 
-OpenEBS is completely free and open source software. There are also commercial options available for organizations seeking enterprise support and management capabilities. These are provided by third-party vendors. For more information, see https://openebs.io <BR>
+
+### Multiple Storage Engines
+OpenEBS is a Kubernetes provides Persistent Stateful Storage Platform that has **5 core Data Engines**<BR>
+Each Storage Engine provides differnt Capabilities, Flexibility, Resilliance, Data Protections, and Performance features.<BR>
+
+| ID  | Data-Eegines       | Where to create your issues                            |
```
Comment (avishnu) [inline README.md:20]: Typo: Engines

File: README.md [hunk 8]
```diff
@@ -1,43 +1,64 @@
...
 <BR>
 
-- Stateful persistent storage volumes for Kubernetes
-- is a 100% Cloud-Native storage solution
-- Delivers a Kubernetes cluster-wide storage fabric that provides containers with access to storage across the entire Kubernetes cluster.
-- Enterprise Grade data management capabilities such as **snapshots, clones and replicated volumes** <BR>
+> [!IMPORTANT]
+> **OpenEBS provides**... <BR>
+> - Stateful persistent Dynamically provisioned storage volumes for Kubernetes
+> - High Performance NVMe-oF storage access optomized for All-Flash Solid State storage media
```
Comment (avishnu) [inline README.md:9]: Typo: optimized

File: README.md [hunk 9]
```diff
@@ -1,43 +1,64 @@
...
+> - is a 100% Cloud-Native storage platform
+> - Delivers a Kubernetes cluster-wide vSAN fabric that provides containers/Pods with resillent access to storage across the entire cluster.
```
Comment (avishnu) [inline README.md:11]: Typo: resilient

File: README.md [hunk 10]
```diff
@@ -1,43 +1,64 @@
...
+> - Enterprise Grade data management capabilities such as **snapshots, clones, replicated volumes, DIskGroups, Volume Groups, Aggregates, RAID** <BR>
```
Comment (avishnu) [inline README.md:12]: DiskGroups instead of DIskGroups

File: README.md [hunk 11]
```diff
@@ -1,43 +1,64 @@
 # Welcome to OpenEBS
 <BR>
-We are an ultra-modern storage target platform, a Hyper-Converged storage software system and a modern Storage Data Fabric natively & tighty integrated into a Kubernetes platform.<BR>
-<BR>
-OpenEBS provides... <BR>
+We are an ultra-modern Block Mode storage platform, a Hyper-Converged software Storage System and a Enterprise Grade virtual NVMe-oF SAN (vSAN) Fabric that is natively & tighty integrated into Kubernetes.<BR>
```
Comment (avishnu) [inline README.md:3]: an Enterprise
tightly

**Comment by avishnu** [comment]
DCO sign-off is missing.

File: README.md (hunk 8 above)
Comment (orville-wright) [reply README.md:9]: fixed

File: README.md (hunk 9 above)
Comment (orville-wright) [reply README.md:11]: fixed

File: README.md (hunk 10 above)
Comment (orville-wright) [reply README.md:12]: Fixed

File: README.md (hunk 7 above)
Comment (orville-wright) [reply README.md:20]: Fixed grammar,

Issue of 1 Single centralized repo was discussed today (4/25/2024) and we agreed on a strategy for this. As noted in issue #3701 and #3709.

File: README.md (hunk 4 above)
Comment (orville-wright) [reply README.md:27]: Fixed

File: README.md (hunk 5 above)
Comment (orville-wright) [reply README.md:28]: Fixed

File: README.md (hunk 6 above)
Comment (orville-wright) [reply README.md:29]: I'd like to leave this in (a is) becasue I want the Maintainer RAW-Device File project to be able to see that RAW-Device File is NOT Archived as a Legacy project. This also shows community users that its not Archived as part of Legacy... and we wont have to answer any questions when they cant find any references to it. 
- I honestly don't think this confuses anyone or hurts the project in any way. It helps clarify things this way.

File: README.md (hunk 2 above)
Comment (orville-wright) [reply README.md:47]: Fixed

File: README.md (hunk 1 above)
Comment (orville-wright) [reply README.md:54]: Fixed. - Sorry about that.

**Comment by orville-wright** [comment]
Merge as all requested fixes have been done.

**Pull request merged**